src/functions/carFunctions.ts

```diff
@@ -0,0 +1,60 @@
+export const checkModel = (str: string) => {
+  if (str === "") throw "Model is empty.";

```

Comment:
I suggest that using if(str.trim() === ""), otherwise, if the user input "  "(some spaces in the string which is a empty string), this if statement cannot detect.